packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -1139,6 +1139,17 @@ new ecs.Ec2Service(this, 'EC2Service', {
 });
 ```
 
+### Cluster Default Provider Strategy
+When the service does not have a capacity provider strategy, cluster's default capacity provider strategy will be used.
+```ts
+new ecs.Cluster(stack, 'EC2CPCluster', {
```

Comment:
```suggestion
declare const capacityProvider: ecs.CapacityProvider;

new ecs.Cluster(stack, 'EC2CPCluster', {
```

Remember, these code snippets should be valid. Make sure to declare any resources whose existence you're implying.